frontend/src/components/Worksheet/GoogleCalendarSync/GoogleCalendarButton.tsx

```diff
@@ -48,9 +50,13 @@ function GoogleCalendarButton({ courses, season_code }: Props): JSX.Element {
       setLoading(false);
     }
     loadGapi();
-  }, [gapi]);
+  }, []);
```

Comment:
@benzuzu I removed this dependency because the only way `gapi` is changed is through this effect, and `gapi`'s  update essentially depends on itself. Please let me know if there's anything I missed.

Reply:
@Josh-Cena Looks good in dev, so def was the right call. Thanks so much!

Reply:
thank you for your service @Josh-Cena